core/kits/manager.php

```diff
@@ -177,6 +177,17 @@ public function get_kit_for_frontend() {
 		return $kit;
 	}
 
+	public function update_kit_settings_based_on_option( $key, $value ) {
+		/** @var Kit $active_kit */
+		$active_kit = $this->get_active_kit();
+
+		if ( $active_kit->is_saving() ) {
+			return;
+		}
+
+		$active_kit->save( [ 'settings' => [ $key => $value ] ] );
```

Comment:
Please try update blogname+description and check how many revisions are created (check post_parent=kit_id)

Reply:
It does not create revisions at all.